docs/revsets.md

```diff
@@ -105,7 +105,13 @@ revsets (expressions) as arguments.
 * `git_head()`: The Git `HEAD` target as of the last import.
 * `visible_heads()`: All visible heads (same as `heads(all())`).
 * `heads(x)`: Commits in `x` that are not ancestors of other commits in `x`.
+  Note that this is different from
+  [Mercurial's](https://repo.mercurial-scm.org/hg/help/revsets) `heads(x)`
+  function, which is equivalent to `x ~ x+`.
 * `roots(x)`: Commits in `x` that are not descendants of other commits in `x`.
+  Note that this is different from
+  [Mercurial's](https://repo.mercurial-scm.org/hg/help/revsets) `roots(x)`
+  function, which is equivalent to `x ~ x-`.
```

Comment:
Maybe equivalent expression is flipped? I think hg's `roots(x)` is `x and not children(x)`.

Reply:
Yes, thanks for catching that.